crates/bevy_reflect/src/serde/mod.rs

```diff
@@ -15,6 +15,10 @@ mod tests {
     };
     use serde::de::DeserializeSeed;
 
+    fn custom_default() -> i32 {
+        -1
+    }
```

Comment:
I would prefer to keep this where it was, duplicated in both test. this function doesn't really make sense as a "global" function, and it reduces readability by making it far away from where it's used